libs/indibase/inditelescope.cpp

```diff
@@ -1384,7 +1384,10 @@ bool Telescope::ISNewSwitch(const char *dev, const char *name, ISState *states,
             IUResetSwitch(&ParkOptionSP);
 
             bool rc = false;
-
+	    // 2020-06-01, wildi, why not writing while parked, slewing, ...?
+	    // In case the driver died, one can not set it to true again,
+	    // without editing the ParkData.xml
+#ifdef no
```

Comment:
Parking option can only be set when unparked and idle. This change might be breaking so we'd avoid it for this release.

Reply:
I understand. The reason why I came up with this is, I had to edit ParkData.xml in order that the mount could be unparked cleanly.